mlrun/common/model_monitoring/helpers.py

```diff
@@ -106,10 +105,21 @@ def _get_edges(hist: Histogram) -> BinEdges:
 
 
 def pad_hist(hist: Histogram) -> None:
-    """Add [-inf, x_0] and [x_n, inf] bins to the histogram inplace"""
+    """
+    Add [-inf, x_0] and [x_n, inf] bins to the histogram inplace unless present
+    """
     counts = _get_counts(hist)
     edges = _get_edges(hist)
 
+    def is_padded() -> bool:
+        factor = 1000
```

Comment:
why do we need the factor, padded histogram should have value +- _MAX_FLOAT at the edges..

Reply:
Right, I was worried about compatibility issues. I checked now and the value we use here:
https://github.com/mlrun/mlrun/blob/b322c4b8dbf1a8ec75bcc390407af5cf429ea90f/mlrun/common/model_monitoring/helpers.py#L30
Is not platform dependent:
https://docs.python.org/3/library/sys.html#sys.float_info

So this can be removed. I'll open a fix to remove this factor.

Reply:
Thanks!

Reply:
https://github.com/mlrun/mlrun/pull/4827